src/components/EditMarkupLink.js

```diff
@@ -13,33 +13,35 @@ import { css } from '@emotion/core';
 import { Code } from '@ndla/icons/editor';
 import { spacing, colors } from '@ndla/core';
 
-const linkStyle = css`
-  box-shadow: none;
-
-  svg {
-    width: ${spacing.normal};
-    height: ${spacing.normal};
-    padding: 2px;
-    border-radius: 50%;
-    border: 2px solid ${colors.brand.light};
-
-    path:last-child {
-      stroke: ${colors.brand.light};
-      fill: ${colors.brand.light};
-    }
+export const EditMarkupLink = ({ title, to, margin }) => {
+  const linkStyle = css`
```

Comment:
Skjønner ikke helt hvorfor du har flyttet denne inn, og endred på fargene på linje 28 og 29?

Reply:
Har ikke forandret fargene! Var bare for å få tilgang til noen props for en conditional sizing for bilde nr 2. Mente jeg reverta filen men ser jeg feilet der!